.github/workflows/benchmark-flow.yml

```diff
@@ -0,0 +1,117 @@
+name: Run a Benchmark Flow
+
+on:
+  workflow_call:
+    inputs:
+      container:
+        type: string
+        default: redisfab/rmbuilder:6.2.7-x64-bullseye
+      module_path:
+        type: string
+        default: bin/linux-x64-release/search/redisearch.so
+      profile_env:
+        type: number # for default of 0
+      cluster_env:
+        type: string # for default of ""
+      benchmark_glob:
+        type: string
+        default: "*.yml"
+      triggering_env:
+        type: string
+        default: "circleci" # TODO: change to "github-actions" when ready on grafana
+      allowed_envs:
+        type: string
+        default: "oss-standalone"
+      allowed_setups:
+        type: string
+      benchmark_runner_group_member_id:
+        type: number
+        default: 1
+      benchmark_runner_group_total:
+        type: number
+        default: 1
+
+jobs:
+  benchmark-steps:
+    runs-on: ubuntu-latest
+    container: ${{ inputs.container }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - run: |
+          git init
+          git config --global --add safe.directory '*'
+          git submodule update --init --recursive
+      - name: Setup
+        run: ./sbin/setup
+      - name: Get Redis
+        uses: actions/checkout@v4
+        with:
+          repository: redis/redis
+          ref: '7.2'
```

Comment:
I think its better use a tag (7.2.1) or unstable branch. Also, why do we need to use a `redisfab` container if we are building redis here? (I would suggest removing `redisfab`)

Reply:
So the redis instance is on the AWS machine and not installed by this flow, correct? I’ll change the reference to `unstable` so we get the latest `redisbench-admin`. We also get the latest unstable JSON as far as I can see from the script

Reply:
correct. The AWS AMI is controlled via the terraform setup specified on the benchmarks spec. And we keep it static over time to avoid any change on the underlying setup that is not really related to the code change on the PR. TLDR, nothing changes except the code of the project.